services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManagerConfig.java

```diff
@@ -31,4 +33,19 @@
 
     @Key("disco-packet-size")
     int getDiscoPacketSize();
+
+    @Key("flow-meter-burst-coefficient")
+    @Default("0.1")
+    @Description("This coefficient is used to calculate burst size for flow meters. " +
+                 "Burst size will be equal to 'coefficient * bandwidth'. " +
+                 "Value '0.1' means that burst size will be equal to 10% of flow bandwidth.")
+    double getFlowMeterBurstCoefficient();
```

Comment:
We should set `min` limit.

Reply:
Unfortunately @Min annotation doesn't work for double and float:

```
 * Note that {@code double} and {@code float} are not supported due to rounding errors
 * (some providers might provide some approximative support).
```

Reply:
Is there some other way to do it?